engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
+
+2. Should user be able to upgrade or downgrade subscriptions?
```

Comment:
I'm not sure how subscriptions and the plans interact so far. From what i remember we set different plans in the config. Does braintree know about these different payment levels?
I think if a provider offers different service levels a user should be able to switch. But for now a single service level and subscribing and unsubscribing should be fine.